### PR TITLE
Added test case BG7

### PR DESCRIPTION
**Pull request by BigGeF**
# Description
Added BG7 test case in readingsBarGroupQuantity.js
Added file expected_bar_group_ri_15-20_mu_kWh_gu_kWh_st_2022-08-20%07#25#35_et_2022-10-28%13#18#28_bd_13.csv

Mason Sain - @sainmas
Evan Thomas - @ethomas5
Pedro Valdovinos Reyes - @PValdovinos

Partly addresses [#692](https://github.com/OpenEnergyDashboard/OED/issues/962)

Fixes #[issue]

## Type of change

- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.org/developer/pr/) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.org/developer/cla/) and each author is listed in the Description section.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1633,14 +1633,6 @@
 				"acorn": "^6.0.0 || ^7.0.0 || ^8.0.0"
 			}
 		},
-		"node_modules/add-dom-event-listener": {
```
Comment (huss) [inline package-lock.json]: I'm curious why there is a change in a package associated with this PR. This PR relates to a test case so this seems outside its scope. Should it be here?

File: src/server/test/web/readingsBarGroupQuantity.js [hunk 2]
```diff
@@ -122,87 +127,113 @@ mocha.describe('readings API', () => {
...
+                        });
                     // Check that the API reading is equal to what it is expected to equal
                     expectReadingToEqualExpected(res, expected, GROUP_ID);
                 });
-                // Add BG7 here
+                
+             
+                    mocha.it("BG7: 13 day bars for 15 + 20 minute reading intervals and quantity units with reduced, partial days & kWh as kWh", async () => {
+                        // Define arrays of data for units, conversions, and test meters/groups
+                        const unitData = unitDatakWh;
```
Comment (huss) [inline src/server/test/web/readingsBarGroupQuantity.js]: I'm not sure it is best to reassign the name of the standard units & conversions to use them. The other test cases have not, as far as I know, done this. For these reasons, I would directly use the name and remove these two lines.

File: src/server/test/web/readingsBarGroupQuantity.js [hunk 3]
```diff
@@ -122,87 +127,113 @@ mocha.describe('readings API', () => {
...
+                            timeInterval: ETERNITY.toString(),
                             barWidthDays: '76',
-                            graphicUnitId: unitId });
+                            graphicUnitId: unitId
+                        });
                     // Check that the API reading is equal to what it is expected to equal
                     expectReadingToEqualExpected(res, expected, GROUP_ID);
                 });
-                // Add BG7 here
+
```
Comment (huss) [inline src/server/test/web/readingsBarGroupQuantity.js]: OED generally leaves at most one blank line between tests. Having two and containing spaces at the front is less desirable. Could you please remove them. There is also two empty blank lines after the test to remove.

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @BigGeF, @sainmas, @ethomas5 & @PValdovinos for their first contribution to OED. Testing and review found it works as expected. I did make a few comments that I would appreciate your response/action.

**Comment by ethomas5** [comment]
@huss Not sure if resolving the changes pinged you on this, but just letting you know we got the PR comments addressed (hopefully).

**Review by huss — APPROVED** [review]
Thanks to the team of @BigGeF, @sainmas, @ethomas5 & @PValdovinos for quickly addressing the comments. Review and testing found that all is in order. Congratulations on your first accepted contribution to OED.

**Pull request merged**